tailwind.config.js

```diff
@@ -5,8 +5,16 @@ export default {
     "./src/**/*.{js,ts,jsx,tsx}"
   ],
   theme: {
-    extend: {},
-  },
+    extend: {
+      colors: {
```

Comment:
**suggestion:** In my reading, it sounds like we can also literally name these colours by name. ie:
      colors: {
          yellow: '#eef26d', //yellow
          lightPurple: '#b796d9', //light purple
          darkPurple: '#4e2973', //dark purple
          white: '#f2f2f2', //white
          black: '#0d0d0d', //black
        }
        
        My thought is this might be more human readable for us as devs and make our work quicker when doing inline styles, instead of needing to pop back and forth and check the numbers.
        
        It looks like in this case, camel case is the suggested naming convention (so that the tailwind kebab-case doesn't get confusing)